### PR TITLE
net-libs/gnome-online-accounts: Bump to 3.50.3

### PR DESCRIPTION
**Pull request by flint2**

### CODE REVIEW

File: net-libs/gnome-online-accounts/metadata.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
   </maintainer>
   <use>
     <flag name="gnome">Use <pkg>gnome-base/gnome-control-center</pkg> to set GNOME online accounts preferences</flag>
+    <flag name="ms-graph">Enable Microsoft 365 provider</flag>
```
Comment (pacho2) [inline net-libs/gnome-online-accounts/metadata.xml:10]: I would name the USE flag "ms365". I think most end-users will understand better what that USE flag is for than understanding " ms-graph" meaning

File: net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# goa kerberos provider is incompatible with app-crypt/heimdal, see
+# https://bugzilla.gnome.org/show_bug.cgi?id=692250
+# json-glib-0.16 needed for bug #485092
+RDEPEND="
+	>=dev-libs/glib-2.67.4:2
+	sys-apps/dbus
+	introspection? ( >=dev-libs/gobject-introspection-0.6.2:= )
+	>=gui-libs/libadwaita-1.4
+	>=gui-libs/gtk-4.10:4
+	>=net-libs/webkit-gtk-2.33.1:4.1
```
Comment (pacho2) [inline net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild:28]: webkit requirement was dropped at 3.49.0 version

File: net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson vala xdg
+
+DESCRIPTION="GNOME framework for accessing online accounts"
+HOMEPAGE="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
```
Comment (pacho2) [inline net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild:9]: Please use https://gitlab.gnome.org/GNOME/gnome-online-accounts as wiki.gnome.org will be disabled soon

File: net-libs/gnome-online-accounts/metadata.xml (hunk 1 above)
Comment (flint2) [reply net-libs/gnome-online-accounts/metadata.xml]: Done in https://github.com/gentoo/gentoo/commit/20171033b2493ec07f0e848030c3200b6e3d66fb

File: net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild (hunk 2 above)
Comment (flint2) [reply net-libs/gnome-online-accounts/gnome-online-accounts-3.50.3.ebuild]: Done in https://github.com/gentoo/gentoo/commit/20171033b2493ec07f0e848030c3200b6e3d66fb

**Pull request closed without merging**